src/Modules/AuthModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,5 +1,95 @@
-import React from "react";
+import { useForm } from "react-hook-form";
+import logo from "../../../../assets/images/PMS 3.png";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useToast } from "../../../../Context/ToastContext";
+import { FormData } from "../../../../interfaces/Auth";
 
 export default function ForgetPass() {
-  return <>ForgetPass</>;
+	const [spinner, setSpinner] = useState<boolean>(false);
+	const { showSuccessToast, showErrorToast } = useToast();
+
+	const {
+		register,
+		handleSubmit,
+		formState: { errors },
+	} = useForm<FormData>();
+	const navigate = useNavigate();
+
+	const onSubmit = async (data: FormData) => {
+		setSpinner(true);
+
+		try {
+			const response = await axios.post(
+				`https://upskilling-egypt.com:3003/api/v1/Users/Reset/Request`,
```

Comment:
why we you don't use baseURL from useContext ?

Reply:
> why you don't use baseURL from useContext ?

and why should we store it in context ? is there a difference between storing it in a context vs constant file that will contain all project URLs ? and if there is a difference what is it ?

Reply:
i asked them to make a folder named with [utils] js file and make all our constans there like BaseURL cause when we use many context it makes so many rendering

Reply:
> Owner

It is not just about rendering, both context and normal files are a way to share code. But the main difference is that we reach for context when we want to share global `state`, and baseURL is not a state. It is just fixed string that will not change during the application at all.